app/assets/stylesheets/govuk_publishing_components/components/_image-card.scss

```diff
@@ -3,6 +3,21 @@
   @include govuk-text-colour;
   position: relative;
   margin-bottom: govuk-spacing(6);
+  display: flex;
+  flex-direction: column-reverse;
+
+  @include govuk-media-query($from: mobile, $until: tablet) {
+    display: block;
+
+    .gem-c-image-card__text-wrapper {
```

Comment:
Something's changed here in the layout on mobile, the gap between the image and the text has increased.

Before | After
------ | ------
<img width="428" alt="Screenshot 2020-09-21 at 08 28 55" src="https://user-images.githubusercontent.com/861310/93741557-92122780-fbe4-11ea-8a83-cf96660b7ec9.png"> | <img width="398" alt="Screenshot 2020-09-21 at 08 29 00" src="https://user-images.githubusercontent.com/861310/93741575-9b9b8f80-fbe4-11ea-8765-019d3f456547.png">
<img width="430" alt="Screenshot 2020-09-21 at 08 29 09" src="https://user-images.githubusercontent.com/861310/93741593-a35b3400-fbe4-11ea-859a-2a18eef935d1.png"> | <img width="390" alt="Screenshot 2020-09-21 at 08 29 17" src="https://user-images.githubusercontent.com/861310/93741618-ad7d3280-fbe4-11ea-8580-7a6874f72171.png">